packages/datagateway-dataview/src/views/card/isis/isisInvestigationsCardView.component.test.tsx

```diff
@@ -115,165 +112,128 @@ describe('ISIS Investigations - Card View', () => {
     jest.clearAllMocks();
   });
 
-  it('renders correctly', () => {
-    const wrapper = createWrapper();
-    expect(wrapper.find('CardView').props()).toMatchSnapshot();
-  });
-
-  it('calls required query, filter and sort functions on page load', () => {
```

Comment:
Is there a reason why these tests were removed? Just curious

Reply:
My reason is they are implementation details so they can break easily. But on that basis I should probably mock axios calls instead of the hook calls too, which I have done for some tests but not others, because I'm not sure whether THAT is also considered implementation details (ie API calls). My feeling is that mocking API calls is not as "implementation details" as mocking hook calls, because hypothetically if we moved on from hooks API to another API, the tests will still pass if we had mocked API calls rather than hook calls. What do you think?

Reply:
I'm fine with moving to mocking axios calls instead - just means more regex path matching. Part of the reason to test these hooks is to test that we're passing the correct params in the query i.e. includes and filters etc. - but I suppose these details are tested in the e2e tests instead and that's probably the better place for it.